README.md

```diff
@@ -49,17 +49,65 @@ There is no return value from this method.
 
 If language selection fails (no matching languages), the callback will contain an error.
 
+
+#### `.transformByField({source}, [languages], {fields});`
+
+This method applies the transform on a field by field basis as opposed to comparing a whole `i18n`
+block like the `transform` method does.
+
+The `fields` object should adhere to the following schema:
+
+```
+{
+    "required": [
```

Comment:
Quotes?

Not sure about the names. What about something like 'fallback' and 'noFallback'?
What if a field is not specified in either list? Can we assume one of the 2 will be the default (e.g. 'Optional?') (to be picky, what if it's in both?).